dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/company.jsp

```diff
@@ -60,289 +67,435 @@ final boolean hasAdminRole = user.isAdmin();
+                     </td>
+                     <td>
+                        <div><a href="#" onclick="imgSwap('/html/images/backgrounds/bg-7.jpg'); dijit.byId('bgPicker').hide();"><img src="/html/images/backgrounds/bg-7-sm.jpg" width="75" height="47"></a></div>
+                     </td>
+                     <td>
+                        <div><a href="#" onclick="imgSwap('/html/images/backgrounds/bg-8.jpg'); dijit.byId('bgPicker').hide();"><img src="/html/images/backgrounds/bg-8-sm.jpg" width="75" height="47"></a></div>
+                     </td>
+                  </tr>
+                  <tr>
+                     <td>
+                        <div><a href="#" onclick="imgSwap('/html/images/backgrounds/bg-9.jpg'); dijit.byId('bgPicker').hide();"><img src="/html/images/backgrounds/bg-9-sm.jpg" width="75" height="47"></a></div>
+                     </td>
+                     <td>
+                        <div><a href="#" onclick="imgSwap('/html/images/backgrounds/bg-10.jpg'); dijit.byId('bgPicker').hide();"><img src="/html/images/backgrounds/bg-10-sm.jpg" width="75" height="47"></a></div>
+                     </td>
+                     <td>
+                        <div><a href="#" onclick="imgSwap('/html/images/backgrounds/bg-11.jpg'); dijit.byId('bgPicker').hide();"><img src="/html/images/backgrounds/bg-11-sm.jpg" width="75" height="47"></a></div>
+                     </td>
+                     <td>
+                        <div><a href="#" onclick="imgSwap(''); dijit.byId('bgPicker').hide();"> <img src="/html/images/backgrounds/bg-no-sm.jpg" width="75" height="47"></a></div>
+                     </td>
+                  </tr>
+               </table>
+            </div>
+         </div>
+         <div style="margin-left: 10rem;">
+            <h3 style="font-weight: normal; margin-bottom: 1rem;">Login Screen Logo</h3>
+            <div class="logo">
+               <button class="logo__delete">&times;</button>
+               <div class="logo__container" style="width: 200px; padding: 1rem; border-radius: 2px;">
+                  <img style="max-width: 200px; width: 100%;" border="1" hspace="0" src="<%= IMAGE_PATH %>/company_logo?img_id=<%= company.getCompanyId() %>&key=<%= ImageKey.get(company.getCompanyId()) %>" vspace="0" />
+               </div>
+            </div>
+            <dot-asset-drop-zone id="dot-asset-drop-zone-main" style="display: none;" drop-files-text="Drop Image" upload-file-text="Uploading Image..." display-indicator="true"></dot-asset-drop-zone>
+            <input type="hidden" name="loginScreenLogoInput" id="loginScreenLogoInput" value="">	
+            <p style="margin-top: 1rem; color: grey;">This is the logo used for the login screen and communications (e.g. emails, etc)</p>
+            <br />
+            <div style="margin-top: 2rem;">
+               <label for="topNav_logo">
+               <input type="checkbox" name="topNav" id="topNav_logo" />
```

Comment:
This needs to be a dojo checkbox.

Reply:
I couldn't make it work. Looking for feedback.